### DISCUSSION TITLE
Compare IDs case sensitive

### DISCUSSION THREAD

**Discussion by MagHem** [PowerShell/PowerShell#17101] [General]
### Summary of the new feature / enhancement

As a user I want that MergeCsv does not treat the content of the id column case-insensitive. An id like "Test" is not the same as "TEST" and therefore it's not a duplicate id!

**Comment by doctordns** [comment]
i am not clear on what you are asking? What did you mean by MergeCSV?  There are not cmdlets that merger CSVs.

Can you be more specific with your query?

**Comment by vexx32** [comment]
It sounds like you're having an issue with a custom script or module that we're not familiar with. The handling in the script or module is down to how it was coded; PowerShell can do case-sensitive comparisons just fine, if you write the code in that way.

I'm going to convert this to a discussion for the time being as it looks like this pertains to something that is external to the repository. I would also highly recommend asking around community channels (e.g., https://aka.ms/psdiscord) for assistance, as the folks there may be more familiar with the module or script you happen to be working with.

<blockquote><img src="https://cdn.discordapp.com/splashes/180528040881815552/4438e80f2285729408fc8ab64921cdc7.jpg?size=512" width="48" align="right"><div><img src="/assets/ec2c34cadd4b5f4594415127380a85e6.ico" height="14"> Discord</div><div><strong><a href="https://discord.com/invite/vFrjnxP">Join the PowerShell Discord Server!</a></strong></div><div>Check out the PowerShell community on Discord - hang out with 8,996 other members and enjoy free voice and text chat.</div></blockquote>

**Comment by kasini3000** [comment]
https://github.com/EliteLoser/MergeCsv  
looks like issue belongs there. It is recommended to add more stars to the powershell project
